setup.py

```diff
@@ -89,10 +88,10 @@ def run(self):
     # requirements files see:
     # https://packaging.python.org/en/latest/discussions/install-requires-vs-requirements/
     install_requires=[
-        "numpy>=1.11.3",
-        "scikit-learn>=0.18",
+        "numpy>=1.20.0",
+        "scikit-learn>=1.0",
         "tqdm>=4.53.0",
-        "pandas>=1.0.0",
-        "termcolor>=1.1.0",
+        "pandas>=1.2.0",
```

Comment:
```suggestion
        "pandas>=1.1.5",
```

Reply:
Should we do this ^ instead?  Since pandas 1.2.0 does not support python 3.7.0 but only > 3.7.1; I'm not sure why that is?